staging/src/k8s.io/api/core/OWNERS

```diff
@@ -1,42 +1,4 @@
 approvers:
-- erictune
-- lavalamp
-- smarterclayton
-- thockin
+- api-approvers
```

Comment:
This adds @liggitt here.

Reply:
covered by `staging/src/k8s.io/api/OWNERS` already

Reply:
Removed.